src/libs/Navigation/OnyxTabNavigator.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {createMaterialTopTabNavigator} from '@react-navigation/material-top-tabs';
+import {withOnyx} from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import PropTypes from 'prop-types';
+import Tab from '../actions/Tab';
+import ONYXKEYS from '../../ONYXKEYS';
+
+const propTypes = {
+    id: PropTypes.string.isRequired,
```

Comment:
Missing propTypes JSDocs. This would be way cleaner with TypeScript because we could just extend/adjust the types exported from react-navigation since that's really what we want to do anyways. I guess in the meantime I'd just create a propTypes declaration for every prop we actually pass in